tests/drum/test_other_cases_per_framework.py

```diff
@@ -311,3 +312,47 @@ def test_set_language(
                 != -1
             )
             assert case
+
+    @pytest.mark.parametrize(
```

Comment:
is this an integration test?  if so, maybe move to tests/integration?  i'm not sure of the test structure here.